domain/account/usecases/get-account.go

```diff
@@ -0,0 +1,23 @@
+package usecases
+
+import (
+	"errors"
+
+	"github.com/Erikaa81/banco-erika/domain/entities"
+)
+
+var (
+	ErrEmptyId = errors.New("the id was not filled")
```

Comment:
```suggestion
	ErrEmptyID = errors.New("the id was not filled")
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms